test/e2e/fixture-builder.js

```diff
@@ -426,6 +428,8 @@ function onboardingFixture() {
           showFiatInTestnets: false,
           showTestNetworks: false,
           useNativeCurrencyAsPrimaryCurrency: true,
+          // TODO(dbrans): Remove this once we remove the petnamesEnabled toggle from the advanced preferences.
```

Comment:
I'm not sure if we have a policy on TODOs but this feels like a natural change that will be easy to spot when petnames becomes enabled by default.

Reply:
Understood. Removing this comment.